src/test/resources/palm-32-fcd-output/part-00000-d4954a88-f4aa-489f-b33a-5c8576b5d885-c000.csv

```diff
@@ -1,2 +1,2 @@
6014.0986	23301.5138	0.0	34530.8164	0.0	125583.7284	6614.0107	31984.9079	{}	{}	{}	{}	{"Converted Production Forest":28600.5448,"Other Utilization Area":81822.5991,"Sanctuary Reserves/Nature Conservation Area":6614.0107,"Production Forest":4848.8827}	{"Rubber plantation":3542.3364,"Secondary forest":24896.0268,"Agriculture":2763.3729,"Oil palm plantation":24398.5485,"Swamp":29043.6031,"Settlements":851.2415,"Grassland/shrub":22752.6953,"Primary forest":8261.1709,"Water bodies":3133.0348,"Mixed tree crops":5941.6982}	{"Bare land":2902.1353,"Mining":1392.3433,"Settlement":5798.0268,"Secondary forest":21966.5842,"Agriculture":9963.5593,"Swamp":7625.7847,"Grassland/shrub":2875.049,"Estate crop plantation":69353.0242,"Body of water":3707.2217}	13342.6717	false	false	false	false	true	true	false	{"2001":23485.8882,"2002":23485.8882,"2003":23374.1445,"2004":23119.9955,"2005":23068.1202,"2006":22694.5486,"2007":22285.1614,"2008":21463.0712,"2009":21006.0263,"2010":20669.6397,"2011":20292.7605,"2012":20136.9792,"2013":19754.7947,"2014":18881.5269,"2015":18606.9329,"2016":18125.0618,"2017":17132.4074,"2018":15378.6605,"2019":15120.9727,"2020":15003.7708,"2021":14849.0634,"2022":14732.6295}	{"2001":31984.9079,"2002":31984.9079,"2003":31984.9079,"2004":31984.9079,"2005":31984.9079,"2006":31984.9079,"2007":31984.9079,"2008":31984.9079,"2009":31984.9079,"2010":31984.9079,"2011":31984.9079,"2012":31984.9079,"2013":31984.9079,"2014":31984.9079,"2015":31984.9079,"2016":31984.9079,"2017":31984.9079,"2018":31984.9079,"2019":31984.9079,"2020":31984.9079,"2021":31984.9079,"2022":31984.9079}	{"2001":6614.0107,"2002":6614.0107,"2003":6614.0107,"2004":6614.0107,"2005":6614.0107,"2006":6614.0107,"2007":6614.0107,"2008":6614.0107,"2009":6614.0107,"2010":6614.0107,"2011":6614.0107,"2012":6614.0107,"2013":6614.0107,"2014":6614.0107,"2015":6614.0107,"2016":6614.0107,"2017":6614.0107,"2018":6614.0107,"2019":6614.0107,"2020":6614.0107,"2021":6614.0107,"2022":6614.0107}	{"2002":365.8928,"2003":306.0243,"2004":425.4469,"2005":782.9588,"2006":1231.4774,"2007":1279.1351,"2008":793.4315,"2009":713.2658,"2010":532.6605,"2011":537.9658,"2012":1255.4523,"2013":1147.8618,"2014":756.465,"2015":1474.5255,"2016":2746.4013,"2017":2011.4348,"2018":374.8897,"2019":271.9092,"2020":271.1414,"2021":244.6289,"2022":225.7991}	{"2002":14169.0317,"2003":14121.4598,"2004":14263.6318,"2005":14457.9871,"2006":14775.3894,"2007":14894.5153,"2008":14537.3838,"2009":14379.8313,"2010":14307.4367,"2011":14243.1121,"2012":14624.1389,"2013":14640.9716,"2014":14369.8445,"2015":14824.2131,"2016":15518.2929,"2017":15018.6597,"2018":14203.1553,"2019":14200.8532,"2020":14201.3905,"2021":14170.9561,"2022":14147.208}	{"2002":223.0242,"2003":201.3516,"2004":10.5289,"2005":1.3833,"2006":4.6881,"2007":4.7649,"2008":1.0759,"2009":2.1519,"2010":3.7659,"2011":3.8427,"2012":4.1501,"2013":2.9204,"2014":1.3065,"2015":14.6794,"2016":75.0879,"2017":63.9436,"2018":3.1509,"2019":0.9222,"2020":0.9222,"2021":0.9222,"2022":0.9222}	{}
```

Comment:
reminder of what we discussed about tests on area with approx and maybe one going over country boundaries

Reply:
Yes, I added two tests, one that shows approx forest loss, and one that mixes area of Brazil and Argentina.